blocked-edges/4.12.54-RHELKernelCephFSCapDropPanic.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.12.54
+# 4.12 before 4.12.54 and 4.11
+from: 4[.](11[.].*|12[.]([1-4]?[0-9]|5[0123]))[+].*
+url: https://issues.redhat.com/browse/COS-2705
+name: RHELKernelCephFSCapDropPanic
+message: "Nodes in clusters running workloads that mount Ceph volumes may experience kernel panics due to a CephFS client bug"
```

Comment:
nit: no need for the quotes, although they're fine if you want to keep them.  You're also missing a trailing period, which we usually include in our messages.

```console
$ for X in blocked-edges/*.yaml; do yaml2json < "${X}" | jq '(.message // "-")[-1:]'; done | sort | uniq -c | sort -n
      1 "z"
      2 "O"
      4 "e"
     12 "n"
    117 "-"
    652 "."
```

Reply:
That's what I get by copying `RHELKernelHighLoadIOWait` files